### PR TITLE
Draft:Multi Object Property

### PR DESCRIPTION
**Pull request by tobiKad**
This is just a part of the Adding Selection of Set Object Property from the main repository.

### CODE REVIEW

File: eventActions.js [hunk 1]
```diff
@@ -538,7 +554,8 @@ ActionSetProp.prototype.toJS = function () {
     return {
         type: this.type,
         refToObjectProperty: this.refToObjectProperty.toJS(),
-        operand: this.operand.toJS()
+        operand: this.operand.toJS(),
+        refsToObjectProperty: this.refsToObjectProperty()
```
Comment (hfinger) [inline eventActions.js]: this should be replaced by something similar like in lines 359-362

File: eventActions.js [hunk 2]
```diff
@@ -525,8 +539,10 @@ ActionSetProp.prototype.reAddEntities = function (entitiesArr) {
  * @returns {ActionSetProp}
  */
 ActionSetProp.prototype.fromJS = function (data) {
+    // FROM JS cost the error and the whole funtionality is broken.
+    // this.refsToObjectProperty.fromJS(data.refsToObjectProperty);
```
Comment (hfinger) [inline eventActions.js]: you need to actually create new instances using "new RefToObjectProperty()".
You can look in lines 339-346 how it is done there for ActionSetElementPropChange... Here you can do the deserialization of a list of instances analogously.

File: eventActions.js (hunk 1 above)
Comment (tobiKad) [reply eventActions.js]: done

File: eventActions.js (hunk 2 above)
Comment (tobiKad) [reply eventActions.js]: done

**Pull request closed without merging**